--- 0/wrapper	2024-08-01 02:26:38.000000000 +0100
+++ 0/wrapper.new	2024-08-23 23:32:58.491909056 +0100
@@ -67,10 +67,10 @@
 fi
 
 # Load winecfg if no arguments given
-APPLICATION=""
-if [ -z "$*" ] ; then
-  APPLICATION="winecfg"
-fi
+#APPLICATION=""
+#if [ -z "$*" ] ; then
+#  APPLICATION="winecfg"
+#fi
 
 # Allow the AppImage to be symlinked to e.g., /usr/bin/wineserver
 if [ -n "$APPIMAGE" ] ; then
@@ -92,7 +92,12 @@
 fi
 
 if [ -z "$APPLICATION" ] ; then
-"$MAIN" "$@"
+    APPLICATION=`find $WINEPREFIX/drive_c -name "notepad++.exe" -print`
+    if [ ! -f "$APPLICATION" ]; then
+        "$MAIN" "$APPDIR/winedata/npp.8.6.9.Installer.x64.exe" &>/dev/null
+    else
+        "$MAIN" "$APPLICATION" "$@" &>/dev/null
+    fi
 else
-"$MAIN" "$APPLICATION"
-fi
\ No newline at end of file
+  "$MAIN" "$APPLICATION" &>/dev/null
+fi
